tools/dvrescue.xsd

```diff
@@ -44,36 +50,180 @@
     <xsd:sequence>
       <xsd:element name="frame" type="frameType" minOccurs="0" maxOccurs="unbounded"/>
     </xsd:sequence>
-    <xsd:attribute name="count" type="xsd:integer"/>
-    <xsd:attribute name="pts" type="xsd:string"/>
-    <xsd:attribute name="end_pts" type="xsd:string"/>
-    <xsd:attribute name="size" type="xsd:string"/>
-    <xsd:attribute name="video_rate" type="xsd:string"/>
-    <xsd:attribute name="chroma_subsampling" type="xsd:string"/>
-    <xsd:attribute name="scanType" type="xsd:string"/>
-    <xsd:attribute name="aspect_ratio" type="xsd:string"/>
-    <xsd:attribute name="audio_rate" type="xsd:integer"/>
-    <xsd:attribute name="channels" type="xsd:integer"/>
+    <xsd:attribute name="scan_type" type="xsd:string">
+      <xsd:annotation>
+        <xsd:documentation xml:lang="en">
+          String representing the interlacement pattern of the frame.
```

Comment:
Can we decide about a list of patterns?

Reply:
@dericed?

Reply:
I suggest T, TT (top field only), B, B, P (progressive).

Reply:
I agree with those and will add them in now. We can adjust later. Why did you put B twice, did you mean something else?

Reply:
B = Bottom Field First
BB = Bottom Field only (weird but in spec...)

Reply:
Haha. Updated, thank you!

Reply:
this works, later this could be an enumeration list